module/Application/src/Module.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @link      http://github.com/zendframework/ZendSkeletonApplication for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Application;
+
+class Module
+{
+    const VERSION = '3.0.0dev';
```

Comment:
:smile:

On the flip side, at least we're moving to only two supported PHP versions for now!